### PR TITLE
feat: show angular version on mouseover

### PR DESCRIPTION
**Pull request by btford**
https://cloudup.com/caIzd8hzcJg

Related to #1

### CODE REVIEW

File: inject.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  var customEvent = document.createEvent('Event');
+  customEvent.initEvent('myCustomEvent', true, true);
+
+  function sendMessage(content) {
+    eventProxyElement.innerText = JSON.stringify(content);
+    eventProxyElement.dispatchEvent(customEvent);
+  }
+
+  sendMessage({ angular: angular.version });
```
Comment (IgorMinar) [inline inject.js]: let's make this `window.angular.version` to be more explicit (and possibly strict-mode safe)

Comment (IgorMinar) [inline inject.js]: we should also check if angular is present at all, in case we are dealing with angulardart

File: inject.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+(function () {
+
+  var eventProxyElement = document.getElementById('__ngDetectorElement');
+
+  var customEvent = document.createEvent('Event');
+  customEvent.initEvent('myCustomEvent', true, true);
```
Comment (IgorMinar) [inline inject.js]: can you call this `angularVersionEvent`?

**Comment by IgorMinar** [comment]
otherwise lgtm

File: inject.js (hunk 2 above)
Comment (btford) [inline inject.js]: yes

File: inject.js (hunk 1 above)
Comment (btford) [inline inject.js]: :+1:

**Comment by btford** [comment]
updated

**Comment by btford** [comment]
:dancers:

**Pull request merged**